tap_stripe/__init__.py

```diff
@@ -374,6 +380,23 @@ def reduce_foreign_keys(rec, stream_name):
                     rec['lines'][k] = [li.to_dict_recursive() for li in val]
     return rec
 
+# Retry 429 RateLimitError 7 times.
+@backoff.on_exception(backoff.expo,
+                        stripe.error.RateLimitError,
+                        max_tries=7,
+                        factor=2)
+def new_request(self, method, url, params=None, headers=None):
```

Comment:
can we name this function properly. **new_request** doesn't give any idea to reader or add comments

Reply:
Renamed `new_rewuest` to `api_request` and added comments.